src/main/java/org/distributed/systems/chord/actors/Node.java

```diff
@@ -11,7 +16,25 @@
     @Override
     public void preStart() throws Exception {
         super.preStart();
-        log.info("Starting up...");
+        log.info("Starting up...     ref: " + getSelf());
+
+        Config config = getContext().getSystem().settings().config();
+        final String nodeType = config.getString("myapp.nodeType");
+        log.info("DEBUG -- nodetype: " + nodeType);
+
+        if(nodeType.equals("regular")){
```

Comment:
What happens when we have a different node type? Shouldn't the central one also do something special?

Reply:
nope, central nodes just waits